examples/with-unsplash/pages/collection/[slug].tsx

```diff
@@ -0,0 +1,32 @@
+import Head from 'next/head'
+import Layout, { siteTitle } from 'components/Layout'
+import { GetServerSideProps } from 'next'
+import Gallery from 'components/Gallery'
+import Collections from 'components/Collections'
+
+export const getServerSideProps: GetServerSideProps = async ({ query }) => {
+  return {
+    props: {
+      query: query
+    }
+  }
+}
```

Comment:
SSR is not needed here, if you only need the query then the component can handle this itself with client side rendering.

Reply:
SSR removed